packages/eventual-send/src/local.js

```diff
@@ -0,0 +1,39 @@
+const q = JSON.stringify;
+
+const ntypeof = specimen => (specimen === null ? 'null' : typeof specimen);
+
+export const localApplyFunction = (t, args) => {
+  if (!(t instanceof Function)) {
+    const ftype = ntypeof(t);
```

Comment:
It was an attempt to make errors more understandable, but not thoroughly tested since we don't yet rely on `applyFunction` in many (if any) places.

I'll adopt the `ftype !== 'function'` test.